src/tests/example.spec.js

```diff
@@ -27,4 +36,73 @@ test.describe('Saucedemo app basic tests', () => {
         await app.shoppingCart.removeCartItemById(0);
         await expect(app.shoppingCart.cartItems).not.toBeAttached();
     });
+
+    test('perform and verify sorting on the Inventory page', async (
+        /** @type {{ app: import('../pages/Application').Application }} */{ app, page },
+    ) => {
+        await app.login.navigate();
+        await app.login.performLogin('standard_user', 'secret_sauce');
```

Comment:
Please, move the login to the beforeEach or beforeAll hook since you're doing it in the each test